lib/std/build/RunStep.zig

```diff
@@ -181,6 +185,9 @@ fn make(step: *Step) !void {
     child.stdout_behavior = stdIoActionToBehavior(self.stdout_action);
     child.stderr_behavior = stdIoActionToBehavior(self.stderr_action);
 
+    if (self.print)
+        printCmd(cwd, argv);
```

Comment:
To me it doesnt make sense to print to stderr, if the process does not return an error code. [Some argument for this behavior](https://unix.stackexchange.com/a/331831).
The user asked explicitly for the output, which should then go to stdout.

Reply:
That link you sent seems to have arguments for both sides, I don't really see consensus.  Because there doesn't seem to be a clear standard on this, I'm hesitant to go against what the rest of zig build is doing without a maintainer (someone who will merge it) blessing such a change.

Reply:
Ok, sounds good to me then. You can mark this as resolved then.